docs/overrides.md

```diff
@@ -46,15 +46,21 @@ It includes the same changes found in the previous file (devise/sessions/new.htm
 
 **decidim-ldap/app/views/layouts/decidim/_wrapper.html.erb**
 
+Override with Deface in **app/overrides/remove_signup_link_in_wrapper.rb**
+
 It modifies the lines 76 to 78 to remove the signup link for ldap enabled organizations.
```

Comment:
Does this comment still apply? should we remove it?

Reply:
This comment is new, maybe we need rewrite it. 
Because the override is still done but with Deface. What do you think?

Reply:
I meant the comment at line 51, I think we can rewrite it, yes